src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryToolCli/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using BinaryTool;
+
+namespace BinaryToolCli;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        if (args.Length < 2 || args.Length > 6)
+        {
+            Console.WriteLine("Usage: BinaryToolCli <target-directory> <output-report-directory> [--allowed-binaries-keep-file <file> | --keep <file>] [--allowed-binaries-remove-file <file> | --remove <file>]");
+            return;
+        }
+
+        var targetDirectory = args[0];
+        var outputReportDirectory = args[1];
+        var allowedBinariesKeepFile = "";
+        var allowedBinariesRemoveFile = "";
+
+        for (int i = 2; i < args.Length; i++)
+        {
+            switch (args[i])
+            {
+                case "--allowed-binaries-keep-file":
```

Comment:
Since not using any cmdline parsing library, should we at least put these in named constants and for instance re-use those in the help string on line 14?
Otherwise these will diverge very fast.